docs/VnVPlan/VnVPlan.tex

```diff
@@ -379,6 +384,95 @@ \subsection{Traceability Between Test Cases and Requirements}
 \wss{Provide a table that shows which test cases are supporting which
 requirements.}
 
+\setlength{\tabcolsep}{1pt}
+\newgeometry{margin=2cm}
+\begin{landscape}
+  \begin{table}[H]
+    \begin{tabular}{|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|}
+      \hline
+      & \textbf{FR1} & \textbf{FR2} & \textbf{FR3} & \textbf{FR4} &
```

Comment:
I'm not sure if there's a better way to do this, but the tables aren't really split in a logical manner (i.e. there's requirements from multiple sections in one table). I was thinking about the organization and there may be some duplicated information if we do it this way. For example if there's a test case TC-1 that tests UH-UP2 and UH-AS1, TC-1 would be duplicated in both tables

Reply:
If one requirement is only ever tested by one TC, maybe a table like this would be better
| Test Case | Requirements Covered |
| -------- | ------- |
| TC1 | R1, R2, R3    |
| TC2 | R4, R5  |
Otherwise, we could keep the one in the commit, but split the requirements in each tables by section instead.

Reply:
yeah that makes more sense, I kinda was just doing what I saw in other old documents, but this is better, I'll change it tomorrow morning